pdfminer/converter.py

```diff
@@ -783,7 +783,7 @@ def render(item: LTItem) -> None:
                 )
                 self.write(s)
             elif isinstance(item, LTFigure):
-                s = '<figure name="%s" bbox="%s">\n' % (item.name, bbox2str(item.bbox))
+                s = '<figure name="{}" bbox="{}">\n'.format(item.name, bbox2str(item.bbox))
```

Comment:
What do you think of using an f-string here as well? 

And in the other locations where `.format()` is used.